packages/gatsby/cache-dir/ensure-resources.js

```diff
@@ -74,6 +74,16 @@ class EnsureResources extends React.Component {
   }
 
   render() {
+    if (!this.state.pageResources) {
+      console.warn(
+        `EnsureResources was not able to find resources for path: "${this.props.location.pathname}"`,
+        `This typically means that an issue occurred building components for that path`
+      )
+      throw new Error(
+        `EnsureResources was not able to find resources for path: "${this.props.location.pathname}"`
+      )
+    }
```

Comment:
I only could get to this error in development so no need to do this on production.
```suggestion
    if (process.env.NODE_ENV !== `production` && !this.state.pageResources) {
      throw new Error(
        `EnsureResources was not able to find resources for path: "${this.props.location.pathname}"
This typically means that an issue occurred building components for that path.
Run \`gatsby clean\` to remove any cached elements.`
      )
    }
```

I would switch it to the above code so we get this message

![image](https://user-images.githubusercontent.com/1120926/75558005-60b3ed80-5a41-11ea-8d36-7e6d1c407074.png)